test/installer/scripts/test_omsadmin.sh

```diff
@@ -1,42 +0,0 @@
-BASE_DIR=$1
-RUBY_TESTVERS=$2
```

Comment:
[omsadmin_systest.rb](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/test/installer/scripts/omsadmin_systest.rb#L60) already covers this test case and more.